cli/integration-test/entrypoint.clj

```diff
@@ -36,24 +37,27 @@
       (future-cancel fut))
     ret))
 
-(defn log-tail []
-  (:out (sh/sh "tail" "-n" "300" "clojure-lsp.integration-test.out" :dir "integration-test/sample-test/")))
+(defn log-tail [file lines]
+  (:out (sh/sh "tail" "-n" (str lines) file :dir "integration-test/sample-test/")))
 
 (def first-print-log-tail?* (atom true))
 
 (defn print-log-tail! []
   (when (medley/deref-reset! first-print-log-tail?* false)
     (binding [*out* *err*]
       (println "--- RECENT LOG OUTPUT ---")
-      (print (log-tail))
-      (println "--- END RECENT LOG OUTPUT ---"))))
+      (print (log-tail "clojure-lsp.integration-test.out" 300))
+      (println "--- END RECENT LOG OUTPUT ---")
+      #_(println "--- LSP TRACE ---")
+      #_(print (log-tail "clojure-lsp.lsp-trace.out" 500))
+      #_(println "--- END LSP TRACE ---"))))
```

Comment:
Any other reason than debugging purpose to have these comments?

Reply:
Correct, when paired with the commented out tracing launcher (in server.clj), these lines assist in debugging. While I'm still debugging the flaky tests, I'd prefer to have this launcher and logging on all the time in integration tests. I tried to add a `:trace-path` setting, similar to the `:log-path` setting, so that I could configure which launcher gets used. But I couldn't get it to work because settings aren't available during launch, for whatever reason. So I left this around to make it easier the next time I want to turn on these traces.